src/tool_settings.rs

```diff
@@ -0,0 +1,30 @@
+use serde::Deserialize;
+use std::fs;
+
+#[derive(Deserialize)]
+pub struct Config {
+    config: ToolsSettings,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct ToolsSettings {
+    game_path: String,
+    cemu_path: String,
+    // add other settings
+}
+
+impl ToolsSettings {
+    pub fn new() -> Self {
+        let file_path = "config/config.toml";
+        let contents =
+            fs::read_to_string(file_path).expect("Should have been able to read the file");
```

Comment:
We might use `if let` syntax here instead, as it allows us to log errors with something like `tracing` instead of just panicking without context. 

That's less important for a config file, but it'll matter a lot for individual game files!

Reply:
I am unsure how to use that, I will try to find